interview-questions.md

```diff
@@ -8,62 +8,67 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: 
+  ---A parameter is a set of instructions or rules for the given code and an argument is a variable that can change to overall outcome of the code.
 
   Researched answer:
-
+----A parameter is a variable in a method definition and the arguments are the data passed into the methods parameters.
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
+  Your answer:The .map method uses two predefined parameters this and new but only this is required.
 
-  Researched answer:
+  Researched answer:The .map method uses two predefined parameters.  Element which is required and this which is optional.
```

Comment:
There are 3 predefined parameters for map().  Value/Element are required and index and array are optional.